src/main/java/domain/Car.java

```diff
@@ -0,0 +1,61 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Car {
+
+    private static final int carMoveBaseline = 4;
+    private static final int validCarNameSize = 5;
```

Comment:
`static final`로 선언된 불변 상수(`constant`)는, 일반 변수와 구분하기 위해 대문자와 언더스코어(_)를 사용하는 `UPPER_SNAKE_CASE`로 작성하는 경우가 일반적인데 가독성을 높이는 차원에서 아래와 같이 수정해보는 건 어떨까요?

```suggestion
    private static final int CAR_MOVE_BASE_LINE = 4;
    private static final int VALID_CAR_NAME_SIZE = 5;
```

Reply:
불변 상수 컨벤션은 생각못했네요! 수정했습니다!